GetStarted/utility.cpp

```diff
@@ -72,11 +72,11 @@ void parseTwinMessage(DEVICE_TWIN_UPDATE_STATE updateState, const char *message)
     json_value_free(root_value);
 }
 
-void SensorInit()
+int SensorInit()
 {
     i2c = new DevI2C(D14, D15);
     sensor = new HTS221Sensor(*i2c);
-    sensor->init(NULL);
+    return sensor->init(NULL);

```

Comment:
>return [](start = 4, length = 6)

not sure how this helps. Is this return code being read by the agent?

Reply:
This change is to know whether or not the sensor setup failed. Yet to see it fail, but this could be a valuable clue if, for example, the send telemetry fails if it cannot read information from the sensor.